docs/standard/net-standard.md

```diff
@@ -1,64 +1,60 @@
 ---
 title: .NET Standard
 description: Learn about .NET Standard, its versions, and the .NET implementations that support it.
-ms.date: 02/13/2020
+ms.date: 10/05/2020
 ms.technology: dotnet-standard
 ms.custom: "updateeachrelease"
 ms.assetid: c044882c-af15-45f2-96d1-534557a5ee9b
 ---
 # .NET Standard
 
-[.NET Standard](https://github.com/dotnet/standard) is a formal specification of .NET APIs that are intended to be available on all .NET implementations. The motivation behind .NET Standard is to establish greater uniformity in the .NET ecosystem. [ECMA 335](https://github.com/dotnet/runtime/blob/master/docs/project/dotnet-standards.md) continues to establish uniformity for .NET implementation behavior, and while ECMA 335 specifies a small set of standard libraries, the .NET Standard specification encompasses a broader range of .NET APIs.
-
-.NET Standard enables the following key scenarios:
-
-- Defines uniform set of BCL APIs for all .NET implementations to implement, independent of workload.
-- Enables developers to produce portable libraries that are usable across .NET implementations, using this same set of APIs.
-- Reduces or even eliminates conditional compilation of shared source due to .NET APIs, only for OS APIs.
-
-The various .NET implementations target specific versions of .NET Standard. Each .NET implementation version advertises the highest .NET Standard version it supports, a statement that means it also supports previous versions. For example, .NET Framework 4.6 implements .NET Standard 1.3, which means that it exposes all APIs defined in .NET Standard versions 1.0 through 1.3. Similarly, .NET Framework 4.6.1 implements .NET Standard 1.4, while .NET Core 1.0 implements .NET Standard 1.6.
+[.NET Standard](https://github.com/dotnet/standard) is a formal specification of .NET APIs that are available on multiple .NET implementations. The motivation behind .NET Standard was to establish greater uniformity in the .NET ecosystem. However, .NET 5 adopts a different approach to establishing uniformity, and this new approach eliminates the need for .NET Standard in many scenarios. For more information, see [.NET 5 and .NET Standard](#net-5-and-net-standard) later in this article.
```

Comment:
We should probably be consistent between ".NET Standard" and "the .NET Standard". Later on in this article you use "the".

Reply:
Agreed.  I'll remove "the"